docs/administration/upgrade/v6.0_changes.adoc

```diff
@@ -73,71 +73,94 @@ https://github.com/openshift/cluster-logging-operator/blob/release-5.9/docs/refe
 
+.Now moved to v6.0 *ClusterLogForwarder* spec.
+[source,yaml]
 ----
-oc adm policy add-cluster-role-to-user collect-application-logs -z logcollector
-oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z logcollector
-----
-
-Additionally, create the following *ClusterRoleBinding* if collecting audit logs:
-
-----
-oc adm policy add-cluster-role-to-user collect-audit-logs -z logcollector
-----
-
-==== Management, Resource Allocation & Workload Scheduling
-Configuration of the management state (i.e. managed, unmanaged), resource request and limits, tolerations, and node selection
-are part of the new ClusterLogForwarder API.
-
-.Previously part of the resource *ClusterLogging* (v5.9)
-[source, yaml]
-----
-apiVersion: logging.openshift.io/v1
-kind: ClusterLogging
+apiVersion: observability.openshift.io/v1  # <1>
+kind: ClusterLogForwarder  # <2>
 metadata:
-  name: instance
+  name: my-forwarder
 spec:
   managementState: Managed
-  collection:
+  collector:  # <3>
     resources:
-      limits: {}
-      requests: {}
-    nodeSelector: {}
-    tolerations: {}
+      limits:
+        cpu: 500m
+      requests:
+        memory: 1Gi
+    nodeSelector:
+      node-role.kubernetes.io/worker: ""
+    tolerations:
+    - key: logging
+      operator: Exists
+...
 ----
-.Now part of v6.0 *ClusterLogForwarder* along with inputs, outputs, filters and pipelines
+Snippet highlights:
+
+<1> API must now be *observability.openshift.io/v1*
+<2> Kind is now the *ClusterLogForwarder* spec
+<3> spec *collector* now includes resources, nodeSelector and tolerations
+
+==== Permissions (Service Accounts)
+
+The *ClusterLogForwarder* now requires a service account, with correct RBAC permissions, to be specified in the configuration.
+
+.Required Service Account
 [source, yaml]
 ----
 apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: instance
+  name: my-forwarder
 spec:
-  managementState: Managed
-  collector:
-    resources:
-      limits: {}
-      requests: {}
-    nodeSelector: {}
-    tolerations: {}
+  serviceAccount:
+    name: my-account
 ...
 ----
+Administrators are required to *explicitly grant log collection permissions* to any service account referenced in the *ClusterLogForwarder*.   There are 3 cluster roles that can be bound to: *collect-application-logs*, *collect-infrastructure-logs* and *collect-audit-logs*.
+
+.ClusterRoleBinding for *my-account*
+----
+oc adm policy add-cluster-role-to-user collect-application-logs -z my-acccount
+oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z my-account
+----
+
+Additionally, if collecting audit logs:
+----
+oc adm policy add-cluster-role-to-user collect-audit-logs -z my-account
+----
+
+.Legacy *openshift-logging*
+If you have all your previous collectors deployed in the namespace *openshift-logging* then you've likely been using the default service account *logcollector*.
```

Comment:
maybe instead of calling 'default' simply say the SA created by earlier versions of the operator